content/applications/general/email_communication/email_template.rst

```diff
@@ -1,98 +1,222 @@
+
+This is not always the case, as some Odoo applications have a link established in the database with
+the website application. In that case, if a specific domain is defined for the websites, the URL
+generated in the email template uses the domain defined on the corresponding website of the company.
 
 .. caution::
    A document shared using the documents application will always use the web.base.url key, as the
    document shared isn't associated with any particular website. Meaning that the URL will always be
-   the same (the web.base.url key value), whatever the company it's shared from, this is a known
-   limitation!
+   the same (the web.base.url key value), no matter what company it's shared from. This is a known
+   limitation.
 
-On the other hand, sales orders made by a customer on one of your Odoo e-commerce websites have a
-link established with the website from which the order was made. As a result, the e-mail sent for
-the sales orders uses the domain name defined for the corresponding website to generate the links.
+On the other hand, sales orders made by a customer, on an Odoo *eCommerce* website, have a link
+established with the website, from which the order was made. As a result, the e-mail sent for sales
+orders uses the domain name defined for the corresponding website to generate the links.
 
-For more information about how to configure your domains, we invite you to check :doc:`our domain name
-documentation </administration/maintain/domain_names>`.
+For more information about how to configure domains, check out :doc:`our domain name documentation
+</administration/maintain/domain_names>`.
 
 Updating translations within email templates
 --------------------------------------------
 
-Email templates are automatically translated. Changing the translations shouldn’t be necessary.
-However, if for a specific reason you’d like to change some of the translations, this can be done.
+In Odoo, email templates are automatically translated. Changing the translations shouldn't be
+necessary. However, if for a specific reason, some of the translations need to be changed, it can be
+done.
+
+However, like any modification in the code, if they aren't done correctly (for example,
+modifications leading to bad syntax), it can break the template, and as a result, the template will
+appear blank.
 
-Like any modification in the code, keep in mind that modifications that aren’t done correctly (for
-example modifications leading to bad syntax) can break the template, as a result, the template
-will appear blank.
+In order to edit translations, follow these steps from the template:
 
-In order to edit your translations, follow these steps from the template.
+#. To access the translations, first enter :ref:`developer mode <developer-mode>`. To enter
+   developer mode, navigate to :menuselection:`Main Odoo dashboard --> Settings`, scroll down to
+   :guilabel:`Developer Tools` section, and click :guilabel:`Activate the developer mode`.
```

Comment:
Again I don't feel it is necessary to restate how to enter dev mode after telling the user to enter it with a reference to instructions, but this is just my opinion :)
```suggestion
#. To access the translations, first enter :ref:`developer mode <developer-mode>`.
```